src/component/mxgraph/BpmnGraph.ts

```diff
@@ -163,9 +163,55 @@ export class BpmnGraph extends mxgraph.mxGraph {
     mxgraph.mxEvent.addMouseWheelListener(throttle(this.createMouseWheelZoomListener(false), config.throttleDelay), this.container);
   }
 
+  /**
+   * @internal
+   */
+  enableNavigation(zoomConfig?: ZoomConfiguration): void {
+    const panningHandler = this.panningHandler;
+
+    // Pan configuration
+    panningHandler.addListener(mxgraph.mxEvent.PAN_START, this.getPanningHandler('grab'));
+    panningHandler.addListener(mxgraph.mxEvent.PAN_END, this.getPanningHandler('default'));
+
+    panningHandler.usePopupTrigger = false; // only use the left button to trigger panning
+    // Reimplement the function as we also want to trigger 'panning on cells' (ignoreCell to true) and only on left-click
+    // The mxGraph standard implementation doesn't ignore right click in this case, so do it by ourselves
+    panningHandler.isForcePanningEvent = (me): boolean => mxgraph.mxEvent.isLeftMouseButton(me.getEvent()) || mxgraph.mxEvent.isMultiTouchEvent(me.getEvent());
+    panningHandler.setPinchEnabled(true);
+    this.setPanning(true);
+
+    // Only set on Graph initizalization
+    if (zoomConfig) {
+      // Zoom configuration
+      this.registerMouseWheelZoomListeners(zoomConfig);
+    }
+  }
+  /**
+   * @internal
+   */
+  disableNavigation(): void {
+    const panningHandler = this.panningHandler;
+    this.setPanning(false);
+    // Remove panningHandler Listeners
+    panningHandler.eventListeners = [];
```

Comment:
**chore:** We have to check if this is the correct way to remove listeners with mxGraph. I don't have much experience on this area. If you have examples, documentation or any insights, this will help my review 😄

Reply:
Thanks for the hint - apparently it wasn't: [https://jgraph.github.io/mxgraph/docs/js-api/files/util/mxEvent-js.html#mxEvent.removeListener](https://jgraph.github.io/mxgraph/docs/js-api/files/util/mxEvent-js.html#mxEvent.removeListener) should be used for that 
-> i have replaced this by the correct removeListener function